tests/test_statsdb_rebuild_security.py

```
@@ -0,0 +1,49 @@
+import json
+import tempfile
+from pathlib import Path
+from swarm.runtime.statsdb.rebuild import StatsDBRebuildMixin
+
+# Create a dummy class that mixes in StatsDBRebuildMixin
+class RebuildTester(StatsDBRebuildMixin):
+    def ingest_events(self, events, run_id):
```

Ruff (0.14.14)

[warning] Unused method argument: `run_id`

(ARG002)